back/engines/commercial/report_builder/app/controllers/report_builder/web_api/v1/reports_controller.rb

```diff
@@ -29,7 +29,10 @@ def create
         def update
           report.attributes = update_params
           side_fx_service.before_update(report, current_user)
-          return send_unprocessable_entity(report) unless report.save
+
+          unless ReportSaver.new(report, current_user).save
```

Comment:
Could we add a test for this behavior?

Reply:
Done https://github.com/CitizenLabDotCo/citizenlab/pull/7427/commits/7d388d6716feedd63a85eeadb5608eda89f94a21